region/lowernorfair/east.json

```diff
@@ -2658,6 +2712,12 @@
                   "heatProof",
                   "canHeatRun"
                 ]},
+                {"obstacle":{
+                  "id": "C",
+                  "requires": [
+                    null
```

Comment:
I don't understand this. Don't some of the requirements account for breaking the obstacle? It might be relevant to go 1 to 4 and back with the multiviola clip.
Or did you figure we should have a separate issue for solving that whole route?

Reply:
You're right we do need another obstacle because of the clip and the return trip.

Reply:
I've made fixes but don't know how to pull the conflict fixes you've made previously so I can add my fixes on top. Might be easier if you just merge into master and I'll add my new stuff in a new PR.